woocommerce/woocommerce
# Added packages to list of contributors.

Include all the contributors, also to sub packages in the contributor list. 
Also, changed the format to HTML to make it easier to use in block editor and switched to use the default branch, for now, and for simplicity.

To test: 

1. create a GH oAuth token via https://github.com/settings/tokens, it needs access to the repo
2. run `grunt contributors`
3. add date (e.g. 2020-03-10)
4. add GH token
5. Nonempty output should be generated into contributors.html
